### DISCUSSION TITLE
is it possible to scan your workpiece?

### DISCUSSION THREAD

**Discussion by dean-ops** [arkypita/LaserGRBL#2007] [Q&A]
is it possible to scan your workpiece to ensure your engraving will be inside the parameters  without making a mark on the workpiece?

**Comment by StuartB4** [comment]
You should have a frame button at the bottom of the main window. That will move the laser around the outside of the image at low power.
![Untitled](https://user-images.githubusercontent.com/16980026/214713144-538a85b4-8c8d-4507-a0b9-08e7975fef71.png)
